asv_bench/benchmarks/index.py

```diff
@@ -16,18 +20,23 @@
 from .config import AsvBenchmarkConfig
 
 
-class Index(AsvBenchmarkConfig):
-    params = (
-        [10 * 1, 10 ** 3],  # values
-        [10 * 1, 10 ** 3],  # partitions
-        [(int, pa.int64()), (str, pa.string())],  # types
-    )
-    param_names = ["number_values", "number_partitions", "dtype"]
+@lru_cache()
+def generate_partition_values(num_partitions):
+    dataset_uuid = uuid.uuid1().hex
+    return [
+        f"{dataset_uuid}/column_1={num_partitions % 10}/column_2={num_partitions % 100}/{uuid.uuid1().hex}.parquet"
```

Comment:
my bad, you should replace
`{num_partitions % 10}` -> `{ix % 10}` since num_partitions is constant